src/Lykke.Logs/Slack/LykkeLogToSlack.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Common.Log;
+using JetBrains.Annotations;
+using Lykke.SlackNotifications;
+
+namespace Lykke.Logs.Slack
+{
+    /// <summary>
+    /// Logs entries to the specified Slack channel. Which types of entries should be logged, can be configured
+    /// </summary>
+    [PublicAPI]
+    public sealed class LykkeLogToSlack : ILog
+    {
+        private readonly ISlackNotificationsSender _sender;
+        private readonly string _channel;
+        private readonly LogLevel _logLevel;
+
+        private LykkeLogToSlack(ISlackNotificationsSender sender, string channel, LogLevel logLevel)
+        {
+            _sender = sender;
+            _channel = channel;
+            _logLevel = logLevel;
+        }
+
+        /// <summary>
+        /// Creates logger with, which logs entries of the given <paramref name="logLevel"/>,
+        /// to the given <paramref name="channel"/>, using given <paramref name="sender"/>
+        /// </summary>
+        public static ILog Create(ISlackNotificationsSender sender, string channel, LogLevel logLevel = LogLevel.All)
+        {
+            return new LykkeLogToSlack(sender, channel, logLevel);
+        }
+
+        public Task WriteInfoAsync(string component, string process, string context, string info, DateTime? dateTime = null)
+        {
+            // ReSharper disable once ImpureMethodCallOnReadonlyValueField
+            if (_logLevel.HasFlag(LogLevel.Info))
```

Comment:
All such checks could be performed in c-tor, so we could init bool fields there and just use this already defined bools here